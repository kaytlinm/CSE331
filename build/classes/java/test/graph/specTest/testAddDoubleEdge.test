#Add Double Edge Test File
#Tests that a graph can be created, two nodes added and two edges going from the same source node to the same
#destination node but with different labels

CreateGraph graph1
AddNode graph1 node1
AddNode graph1 node2
AddEdge graph1 node1 node2 edge1
AddEdge graph1 node1 node2 edge2
