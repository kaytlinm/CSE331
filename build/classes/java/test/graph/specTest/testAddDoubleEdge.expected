#Add Double Edge Test File
#Tests that a graph can be created, two nodes added and two edges going from the same source node to the same
#destination node but with different labels

created graph graph1
added node node1 to graph1
added node node2 to graph1
added edge edge1 from node1 to node2 in graph1
added edge edge2 from node1 to node2 in graph1